uint8 status_AorM
uint8 status_gear
uint8 status_speed
int16 status_steer
uint16 status_brake
int32 status_enc